src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/WindowsRuntime/EventArgsMarshalerSupport.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.StubHelpers;
+
+namespace Internal.Runtime.InteropServices.WindowsRuntime
+{
+    public static class EventArgsMarshalerSupport
+    {
+        public static IntPtr CreateNativeNCCEventArgsInstance(int action, object newItems, object oldItems, int newIndex, int oldIndex)
```

Comment:
Does this really need to be in the runtime? It just creates an instance of a WinRT type...

Reply:
The problem is  that WindowsRuntimeMarshal.GetActivationFactory() API takes Type as argument. For this case, type(PropertyChangedEventArgs) is a projected type  and in managed side, you can't use its WinRT Type.

maybe S.P.Corelib could expose a public implementation API (GetActivationFactory) which takes a string as argument.

Reply:
Okay. move these marshaling into S.R.WR